client/src/components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.Foot {
```

Comment:
ps: the class names still need to be changed from Foot & FootContainer to Head & HeadContainer